### PR TITLE
Pass scopes to gcloud auth command when setting up credentials for the first time

### PR DESCRIPTION
**Pull request by twindebank**
### Description

Means folks shouldn't have to run any [additional auth commands](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#local-oauth-gcloud-setup) when using dbt with BigQuery for the first time.

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-bigquery next" section.

### CODE REVIEW

File: dbt/adapters/bigquery/gcloud.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
-from dbt.events import AdapterLogger
+from typing import Optional, Tuple
+
 import dbt.exceptions
-from dbt.clients.system import run_cmd
+from dbt.clients import system
+from dbt.events import AdapterLogger
```
Comment (jtcohen6) [inline dbt/adapters/bigquery/gcloud.py:5]: - Are you using anything from `dbt.clients.system` besides `run_cmd`? Or is this a preference for namespacing `system.run_cmd` where it's called below?
- Did you end up needing `AdapterLogger`? If not, could you remove the import?

File: tests/unit/test_bigquery_gcloud.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        call('.', ['gcloud', '--version']),
+        call('.', ['gcloud', 'auth', 'application-default', 'login']),
+    ]
+
+
+def test_setup_default_credentials_with_scopes(run_cmd):
+    setup_default_credentials(scopes=['scope1', 'scope2'])
+    assert run_cmd.call_args_list == [
+        call('.', ['gcloud', '--version']),
+        call('.', ['gcloud', 'auth', 'application-default', 'login', '--scopes', 'scope1,scope2']),
```
Comment (jtcohen6) [inline tests/unit/test_bigquery_gcloud.py:27]: 👍 thanks for adding the test!

**Review by jtcohen6 — COMMENTED** [review]
@twindebank Very neat! Thanks for the contribution, and so sorry for the delay getting back to you. We'd be happy to include this in v1.1. Just one comment after taking a look at the code.

**Comment by twindebank** [comment]
@McKnight-42 done: 
image

**Pull request closed without merging**